templates/solutions/ai/index.html

```diff
@@ -0,0 +1,571 @@
+        <hr />
+        <a class="p-button--positive" href="/contact-us" aria-controls="ai-modal">Contact Canonical</a>
+      </div>
+      <div class="col">
+        <div class="u-embedded-media">
+          <iframe class="u-embedded-media__element" width="560" height="315" src="https://www.youtube.com/embed/yerW2pOczvA?si=arG1DJW8pqHzmVNU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Why Canonical <br />for enterprise AI?</h2>
+    </div>
+    <div class="col">
+      <ul class="p-list--divided">
+        <li class="p-list__item is-ticked">Run your entire AI/ML lifecycle on a single integrated stack</li>
+        <li class="p-list__item is-ticked">Control your TCO with predictable costs and the best economics in the industry</li>
+        <li class="p-list__item is-ticked">Consume open source AI software at speed, securely</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>One integrated AI stack</h2>
+    </div>
+    <div class="col">
+      <p>With Canonical, you get a full open source AI solution and one vendor to support it.</p>
+      <p>From GPUs that are optimised for AI use cases to a desktop environment, an integrated MLOps platform, and scalable edge deployments &mdash; do it all on a single stack.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Fast-track compliance <br />with secure AI software</h2>
+    </div>
+    <div class="col">
+      <p>Rapidly access all the open source software you need for AI, and run it securely.</p>
+      <p>We secure and support all of the key open source software in the AI lifecycle so you don't have to. We fix Critical CVE vulnerabilities in under 24h on average, helping you streamline compliance and focus on your AI models.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Predictable costs and optimal TCO</h2>
+    </div>
+    <div class="col">
+      <p>No software licence fees and predictable pricing per node.</p>
+      <p>Enjoy the freedom of open source, with the best support and economics in the industry.</p>
+      <hr />
+      <p>
+        <a href="/contact-us" aria-controls="ai-modal">Get in touch to kickstart your journey&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width" />
```

Comment:
Use of `hr` with `is-fixed-width` is not recommended, not that we have `p-rule`. It's better to put `p-rule` inside `u-fixed-width`.

Reply:
That's what I usually do, but then I noticed in [the docs](https://vanillaframework.io/docs/base/separators#horizontal-rule) it mentions using `<hr class="is-fixed-width">` as an alternative to nesting the hr inside a row which I assumed is essentially the same thing as nesting it inside `u-fixed-width`. If that's not the case though, what's the correct usage of `<hr class="is-fixed-width">` ?

Reply:
Good question, I guess we need to have a look at our docs (p-rule vs hr) and see if there are any inconsistencies or old recommendations.

I believe that `is-fixed-width` is basically and old recommendation.

The main downside of `hr` (without `p-rule`) is that it adds unnecessary margin. 
It is possible that `p-rule is-fixed-width` would also work, but I don't think we tested that combination directly.

Overall, it's not a huge issue. It's not like we are deprecating or removing non-rule `hr`s. So if `is-fixed-width` is useful (to avoid additional markup), I guess it can stay.

I'll try to have a look at the docs and make sure they express the way we want those to be used now. Feel free to leave it as is here.